user_mail_server/res_users.py

```diff
@@ -17,35 +17,31 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
-from openerp import models, fields, api, _
-from openerp.exceptions import Warning
 import logging
+
+from odoo import models, fields, api, _
+from odoo.exceptions import Warning
+from passlib.hash import sha512_crypt
+
 _logger = logging.getLogger(__name__)
 
-class res_users(models.Model):
+
+class ResUsers(models.Model):
     _inherit = 'res.users' 
 
     @api.model
     def create(self, values):
-		#_logger.info("In create serverside::::::::::::::")
-		ctx = self.env.context.copy()
-		ctx.update({'no_reset_password' : True})
-		return super(res_users, self.with_context(ctx)).create(values)
-
-    @api.one
-    def write(self,values):
-        #raise Warning(values)
-        if values.get('password') and not values.get('dovecot_password',False):
+        ctx = self.env.context.copy()
+        ctx.update({'no_reset_password': True})
+        return super(ResUsers, self.with_context(ctx)).create(values)
+
+    def write(self, values):
```

Comment:
Question: Won't removing api.one have implications we have to handle here?

Reply:
I dont think it will be an issue, @api.multi has been added